.travis.yml

```diff
@@ -4,21 +4,28 @@ addons:
 services:
   - postgresql
 
-before_script:
-  - psql -c 'create role simplified_test;' -U postgres
-  - psql -c 'create database simplified_content_test;' -U postgres
-  - psql -c 'grant all privileges on database simplified_content_test to simplified_test;' -U postgres
-
 language: python
 
 python:
   - "2.7"
 
+cache: pip
+
+git:
+  submodules: false
```

Comment:
I don't understand this, given that we are using a submodule. Is this necessary given that the submodule configuration is already in .gitmodules?